lib/types/protocol.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package common
+package types
 
 type Protocol struct {
```

Comment:
they are also fields used in the backend models - so living in api doesn't feel right.